wpeverest/everest-forms
# Feature - Email preview

### All Submissions:



### Changes proposed in this Pull Request:

Previously, there was no option to preview the email message whenever a new email connection was created. Now, we have added the option to preview the email message using both with and without an email template.


### How to test the changes in this Pull Request:

1. Go to Form Settings -> Email
2. Click on the eye icon (if your email connection is already saved otherwise Click on the Save button before previewing)
3. Check if the correct template is being loaded for the email message or not.

### Types of changes:



### Other information:



### Changelog entry

> Feature - Email preview
